spec-grpc/pom.xml

```diff
@@ -69,78 +79,42 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java-util</artifactId>
+            <scope>provided</scope>
+        </dependency>
     </dependencies>
 
     <build>
-        <extensions>
-            <!--
-                Seems to be needed by the protobuf-maven-plugin.
-                Adding these extensions to the profile is not possible
-            -->
-            <extension>
-                <groupId>kr.motd.maven</groupId>
-                <artifactId>os-maven-plugin</artifactId>
-                <version>${os-maven-plugin.version}</version>
-            </extension>
-        </extensions>
+        <plugins>
+            <!-- Protocol Buffers Compiler Plugin -->
+            <plugin>
+                <groupId>io.github.ascopes</groupId>
+                <artifactId>protobuf-maven-plugin</artifactId>
+                <version>4.0.2</version>
+                <configuration>
+                    <skip>${skip.protobuf.generate}</skip>
```

Comment:
shouldn't that be put in a profile?

Reply:
I see. I prefer to use Maven profiles to control that kind of behaviour but up to you